sci-misc/boinc/boinc-7.8.6.ebuild

```diff
@@ -56,7 +56,7 @@ PATCHES=(
 	# >=x11-libs/wxGTK-3.0.2.0-r3 has webview removed, bug 587462
 	"${FILESDIR}"/fix_webview.patch
 	# xlocale.h was removed in modern glibc, bug 639108
-	"${FILESDIR}"/fix_xlocale.patch
+	"${FILESDIR}"/${MY_PV}-fix_xlocale.patch
```

Comment:
Yes. The reason to do it so is, that boinc-7.9.x is part of the 7.10 series. No idea why. And that the patch is generally working for the whole series without changing.

My consideration was, that I would otherwise have to change from ${P}-fix_xlocale.patch to ${PN}-7.9.3-fix_xlocale.patch on the next version bump. (As it can be seen in a lot of ebuilds) or I'd have to copy a patch that almost never changes.

Previously that patch, like the fix_webview patch, was without version. But with 7.10 it no longer fit, so I had to introduce a version and thought the series would be best.

And the devmanual says, in its very first sentence under "Patches":

> There is no fixed rule for patch naming.

So I thought it was okay to do so...

Reply:
Well, it *is* ok. ;) In the end it comes down to the package maintainer's preference. I usually pick the precise version for the package the patch was introduced with, and as you said change to ${PN}-x.x.x on bumps.